frontend/components/Domain/Recipe/RecipeDataTable.vue

```diff
@@ -3,6 +3,8 @@
     v-model="selected"
     item-key="id"
     show-select
+    sort-by="dateAdded"
+    sort-desc
```

Comment:
Yeah it was screwy for one of the tables (I think it was the foods table) so I went and added an explicit sort to all of them, to prevent any issues in the future

Reply:
Yeah, I wrote this comment before I saw the rest of them 👍